src/elements/Select.tsx

```diff
@@ -155,8 +152,8 @@ const SmallSelectContainer = styled.div.attrs<SelectProps>({})`
     cursor: pointer;
     pointer-events: none;
     position: absolute;
-    top: 12px;
-    margin-left: 8px;
+    top: 10px;
+    margin-left: -8px;
```

Comment:
Just realized, we also need to account for the width of the select element as well

Reply:
@l2succes - can you elaborate? We can probably get away with not having to use JS measurements and rely only on CSS, yah? (This will also need to be SSR capable)